mobile-first/css/index.css

```diff
@@ -0,0 +1,166 @@
+/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+	box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */	
+
+/* Mobile First Styles */
+.container {
+        width: 500px;
+        margin: 0 auto;
+    }
+
+.header-image{
+        display: none;
+    }
+
+.navigation {
+        display: flex;
+        flex-wrap: nowrap;
+        align-items: center;
+        text-align: center;
+        flex-direction: column;
+        margin: 30px 20px;
+	    width: 500px;
+}
+
+.navigation a{
+	color: #a2a5a6;
+    text-decoration: none;
+    border-bottom: 1px solid black;
+    width: 540px;
+    margin: 10px 175px;
+    font-size: 20px;
+    padding-bottom: 20px;
+}
+
+.navigation .logo{
+        order: -1;
+        margin: 10px 230px 10px 230px;
+        padding: 0 180px 20px 180px;
+        border-bottom: 1px solid black;
+    }
+
+header {
+    width: 500px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.main-heading {
+   font-size: 70px; 
+   text-align: center;
+}
+
+.get-started {
+    font-size: 20px;
+	width: 200px;
+    height: 40px;
+	border: 1px solid black;
+	background-color: white;
+    position: relative;
+}
+
```

Comment:
When it comes to buttons, sometimes it's easier to just use a generic div so you don't have to undo all the default styling.